Doc/library/sqlite3.rst

```diff
@@ -406,14 +406,22 @@ Connection Objects
 
    .. attribute:: isolation_level
 
-      Get or set the current default isolation level. :const:`None` for autocommit mode or
-      one of "DEFERRED", "IMMEDIATE" or "EXCLUSIVE". See section
-      :ref:`sqlite3-controlling-transactions` for a more detailed explanation.
+      Get or set the current default isolation level.
+      Set to :const:`None` to disable implicit transaction handling,
+      or one of "", "DEFERRED", "IMMEDIATE" or "EXCLUSIVE".
```

Comment:
```suggestion
      or to one of the ``"DEFERRED"``, ``"IMMEDIATE"`` or ``"EXCLUSIVE"``
      transaction-handling modes.
```

Parallel the first clause of the sentence in stating the meaning of these values, and save the mention of the alias for `DEFERRED` until when the default is introduced, to avoid confusion. Also, use correct reST syntax.